js/background.js

```diff
@@ -122,3 +122,24 @@ browser.runtime.onMessage.addListener(({ type, data }, sender) => {
 
   return null
 })
+
+// =============================================================================
+// Context Menu Logic
+
+const sidebartabs = browser.menus.create({
+  title: 'Add to Sidebar',
+  contexts: ['page'],
```

Comment:
So, if I right click in a blank part of the page, I can add the page to the sidebar, but if I right click on the tab I cannot. If I right click on selected text I cannot.

Also, shouldn't I be able to right click on a link to add it to the sidebar? That was what I thought this was itnened to fix at first.

Please review the valid context menu types: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/menus/ContextType

Reply:
Hmm the link adding to sidebar sounds like a cool addition. I thought it was the page itself but will change to add both.